client/src/components/HeaderImage.js

```diff
@@ -0,0 +1,10 @@
+export default function HeaderImage() {
+    return (
+        <img 
+            clasName="headerImage" 
+            src="../images/mental-health-tracker-headerImg.jpg" 
```

Comment:
At the moment this isn't actually rendering:
![image](https://user-images.githubusercontent.com/78568904/176031918-2d9419ec-5cbe-4d79-802c-607ab6799e86.png)

This I believe is due to some Webpack (the bundling used by create-react-app) strangeness where images stored locally must first be imported:
```
import image from 'path'

<img src={image}/>